src/main/java/com/st/maven/debian/DebianPackageMojo.java

```diff
@@ -208,6 +213,14 @@ public void execute() throws MojoExecutionException {
 		}
 	}
 
+	private String composeInstallDir() {
+		if (installDir != null && !installDir.trim().isEmpty()) {
+			installDir = installDir.startsWith("/") ? installDir.substring(1) : installDir;
+			return installDir.endsWith("/") ? installDir + project.getArtifactId() : installDir + "/" + project.getArtifactId();
+		}
+		return "home/" + unixUserId + "/" + project.getArtifactId();
```

Comment:
previously installDir was absolute directory. I.e. "/home/..." After the change it will be relative, so no longer valid directory. As a relative directory it is used only once during tar build. Can you make it relative only there?

Reply:
Got it, fixed